build.gradle

```diff
@@ -855,8 +855,44 @@ task prepareAsciidocBuild(type: Sync) {
 	into "$buildDir/asciidoc"
 }
 
-task('makePDF', type: org.asciidoctor.gradle.AsciidoctorTask) {
+task checkAsciidocLinks {
 	dependsOn prepareAsciidocBuild
+	inputs.dir("$buildDir/asciidoc")
```

Comment:
Why do you do this in `buildDir`, but not in the `src/reference/asciidoc/` ?

Reply:
Why does it matter? The `prepareAsciidocBuild` copies them there and all other doc tasks work on that dir.

Reply:
Got it. Thanks!